src/BigQuery/BigQueryClient.php

```diff
@@ -80,9 +80,8 @@ class BigQueryClient
      *           requests specifically for authentication.
      *     @type callable $httpHandler A handler used to deliver Psr7 requests.
      *           Only valid for requests sent over REST.
-     *     @type string $keyFile The contents of the service account
-     *           credentials .json file retrieved from the Google Developers
-     *           Console.
+     *     @type array $keyFile The contents of the service account credentials
```

Comment:
I know the type says array, but a bit of added description making sure people know it has to be an associative array and not an object might be smart.. So `json_decode($path, true)`, not `json_decode($path)`.